api/src/test/java/org/openmrs/module/registrationcore/RegistrationCoreUtilTest.java

```diff
@@ -1,13 +1,48 @@
 package org.openmrs.module.registrationcore;
 
 import org.joda.time.DateTime;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Context.class})
 public class RegistrationCoreUtilTest {
-
+	
+	private AdministrationService administrationService;
+	
+	@Before
+	public void setUp() {
+		PowerMockito.mockStatic(Context.class);
+		when(Context.getAdministrationService()).thenReturn(mock(AdministrationService.class));
+		
+		administrationService = Context.getAdministrationService();
+		when(administrationService.getGlobalProperty(RegistrationCoreConstants.GP_BIRTHDATE_ESTIMATION_START_MONTH)).thenReturn("");
+	}
+	
+	@Test
+	public void calculateBirthdateFromAge_shouldUseGPMonthWhenCalculateProperBirthdateWhenYearsSpecified() {
```

Comment:
Am trying to understand the method name's part which reads "..WhenCalculateProperBirthdateWhenYearsSpecified"